index.js

```diff
@@ -115,7 +115,7 @@ client.on("messageCreate", async message => {
     let formattedUserMessage = `${message.author.username} (${await getPronouns(message.author.id)}) on ${DateTime.now().setZone('utc').toLocaleString(DateTime.DATETIME_FULL)}: ${message.content}\n${imageDetails}`;
     if (message.reference) {
       await message.fetchReference().then(async (reply) => {
-        if (reply.author.id == "954288870244114473") {
+        if (reply.author.id == "954288870244114473") {  // FIXME
```

Comment:
what?

Reply:
make that not static

Reply:
i cant remember what that's for?

Reply:
#35

Reply:
OH wait why tf is it hardcoded

Reply:
laziness

Reply:
![image_proxy](https://github.com/spongedsc/SpongeChat/assets/111113093/7849b895-7a4a-4f80-bac9-6adc22fc24fa)